### ISSUE TITLE
don't see any impovement

### ISSUE THREAD

**Issue by sunilit42** [magesuite/magepack#29]
As i see it is calling two js per page + merge js as well

does this behaviour right?

**Comment by sunilit42** [comment]
Hello,

for me 3 js calling
1) common
2) home 
3) js merge 

i think js merge should be not call

**Comment by Januszpl** [comment]
Yes. Check this answer. https://github.com/magesuite/magepack-magento/issues/2
